node/miden-node.toml

```diff
@@ -17,5 +17,5 @@ store_url = "http://localhost:28943"
 [store]
 # port defined as: sum(ord(c)**p for (p, c) in enumerate('miden-store', 1)) % 2**16
 endpoint = { host = "localhost", port = 28943 }
-database_filepath = "db/miden-store.sqlite3"
+database_filepath = "database/miden-store.sqlite3"
```

Comment:
Because we added `db/` as a prefix to the file path of the db files we needed to add `db/` to the `.gitignore` this caused problems with editors and git in general with the other `db` folder that we have in the `store` crate.